test/e2e-go/features/incentives/payouts_test.go

```diff
@@ -211,14 +215,21 @@ func TestBasicPayouts(t *testing.T) {
 		a.NoError(err)
 
 		for _, c := range []libgoal.Client{c15, c01, relay} {
+			_, err := c.WaitForRound(status.LastRound)
+			a.NoError(err)
 			data, err = c.AccountData(block.Proposer().String())
 			a.NoError(err)
-			a.Equal(block.Round(), data.LastProposed)
-			a.Equal(pdata, data)
```

Comment:
Here we're comparing the entire account struct, so I had to pick out individual fields